vertx-pg-client/src/test/java/io/vertx/pgclient/tck/PgConnectionAutoRetryTest.java

```diff
@@ -42,6 +42,7 @@ public void tearDown(TestContext ctx) {
   protected void initialConnector(int proxyPort) {
     SqlConnectOptions proxyOptions = new PgConnectOptions(options);
     proxyOptions.setPort(proxyPort);
+    proxyOptions.setHost("localhost");
```

Comment:
can you update other auto retry tests(they all extends ConnectionAutoRetryTestBase) in this project as well?

Reply:
The proxy server listening on a default host also seems not reliable for the testing, can we try to get rid of it and use the database host instead?

Reply:
Each *ConnectionAutoRetryTest uses `proxyOptions.setHost("localhost")` now.

Reply:
The proxy server uses `this.netServer.listen()`, this is listening on default host = "0.0.0.0" = all interfaces.
I don't see how this could be unreliable.
Vert.x always starts a NetServer on localhost. For starting a server running at some other address we need Docker to create that server on a docker network. This is a different feature and should be tested in a different test, not in the connection auto retry test.